Code/GraphMol/FindStereo.cpp

```diff
@@ -367,72 +380,111 @@ std::string getBondSymbol(const Bond *bond) {
 }
 
 namespace {
-std::string getAtomCompareSymbol(const Atom &atom) {
-  auto fmt = boost::format("%d%s") % atom.getIsotope() % atom.getSymbol();
-  return fmt.str();
+inline std::string getAtomCompareSymbol(const Atom &atom) {
+  // we originally tried this with boost::format, but it was WAY slower
+  return std::to_string(atom.getIsotope()) + atom.getSymbol() +
+         std::to_string(atom.getFormalCharge());
 }
 }  // namespace
 
-std::vector<StereoInfo> findPotentialStereo(ROMol &mol, bool cleanIt,
-                                            bool flagPossible) {
-  std::map<int, Atom::ChiralType> ochiralTypes;
-
-  if (!mol.getRingInfo()->isInitialized()) {
-    MolOps::symmetrizeSSSR(mol);
-  }
-
-  boost::dynamic_bitset<> knownAtoms(mol.getNumAtoms());
-  boost::dynamic_bitset<> possibleAtoms(mol.getNumAtoms());
-  std::vector<std::string> atomSymbols(mol.getNumAtoms());
+namespace {
+void initAtomInfo(ROMol &mol, bool flagPossible, bool cleanIt,
+                  boost::dynamic_bitset<> &knownAtoms,
+                  std::vector<std::string> &atomSymbols,
+                  boost::dynamic_bitset<> &possibleAtoms) {
   for (const auto atom : mol.atoms()) {
     auto aidx = atom->getIdx();
+    atomSymbols[aidx] = getAtomCompareSymbol(*atom);
     if (detail::isAtomPotentialStereoAtom(atom)) {
       auto sinfo = detail::getStereoInfo(atom);
       switch (sinfo.specified) {
         case Chirality::StereoSpecified::Unknown:
+          knownAtoms.set(aidx);
+          atomSymbols[aidx] += std::to_string(aidx);
+          break;
         case Chirality::StereoSpecified::Specified:
           knownAtoms.set(aidx);
+          if (sinfo.descriptor == StereoDescriptor::Tet_CCW) {
+            atomSymbols[aidx] += "_CCW";
+          } else if (sinfo.descriptor == StereoDescriptor::Tet_CW) {
+            atomSymbols[aidx] += "_CW";
+          } else {
+            atomSymbols[aidx] += "_@";
```

Comment:
This suffix only means that this isn't either CCW or CW, right? I guess we will update this when we add the other chirality types. Should we use another symbol rather than `@`? My thought is that SMILES uses `@` as the indicator for CCW centers, and someone might get confused by seeing it here.